File: wscodeblocks/hfsguardadir-cpp/hfsguardadir/catalogador/RotuloRaiz.cpp
#include "RotuloRaiz.h"

namespace hfsguardadir {

namespace catalogador {

RotuloRaiz::RotuloRaiz()
{

}

std::string RotuloRaiz::getRotulo() {
    return rotulo;
}

void RotuloRaiz::setRotulo(const std::string &rotulo) {
    this->rotulo = rotulo;
}

}
}

File: HFSGuardaDiretorioVS2022/HFSGuardaDiretorio_VBNet/catalogador/RotuloRaiz.vb
'
' * Created by SharpDevelop.
' * User: henrique.souza
' * Date: 07/07/2015
' * Time: 11:01
' * 
' * To change this template use Tools | Options | Coding | Edit Standard Headers.
' 

Namespace catalogador
	''' <summary>
	''' Description of RotuloRaiz.
	''' </summary>
	Public Class RotuloRaiz
		Private m_rotulo As String

		Public Sub New()
		End Sub

		Public Property Rotulo() As String
			Get
				Return m_rotulo
			End Get
			Set
				m_rotulo = value
			End Set
		End Property

	End Class
End Namespace
